### PR TITLE
service: add Spanish translation

### PR DESCRIPTION
**Pull request by maegop**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/linux/service.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# service
+
+> Gestiona los servicios mediante la ejecución de scripts init.
+> Se debe omitir la ruta completa del script (se asume `/etc/init.d/`).
+
+- Lista el nombre y el estado de todos los servicios:
+
+`service --status-all`
+
+- Iniciar/Parar/Reiniciar/Recargar servicio (iniciar/parar debería estar siempre disponible):
```
[the remark this answers is not in the document]
Comment (navarroaxel) [reply pages.es/linux/service.md:10]: @maegop you marked this as `resolved`, but you didn't send the changes :upside_down_face:

[the remark this answers is not in the document]
Comment (maegop) [reply pages.es/linux/service.md]: Sorry for the mistake @navarroaxel

**Review by navarroaxel — APPROVED** [review]

File: pages.es/linux/service.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (marchersimon) [reply pages.es/linux/service.md]: @navarroaxel why the underscores?

[the remark this answers is not in the document]
Comment (navarroaxel) [reply pages.es/linux/service.md]: Because it's an English word, it's common use italics.

[the remark this answers is not in the document]
Comment (marchersimon) [reply pages.es/linux/service.md]: But I don't think that any client supports rendering italic text and we don't use it in any other pages.

[the remark this answers is not in the document]
Comment (navarroaxel) [reply pages.es/linux/service.md]: If it's not supported the words are still readable. pages like `supervisorctl` and `csvcut` have `**` and we have no issues with them.

**Review by CleanMachine1 — APPROVED** [review]

**Pull request merged**